sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -41,18 +39,20 @@ public static DirectConnectionConfig getDefaultConfig() {
         return new DirectConnectionConfig();
     }
 
-    //  TODO: (DANOBLE) - To update these docs.
-
     /**
-     * Gets the direct connection timeout
+     * Gets the connection timeout for direct client,
+     * represents timeout for establishing connections with an endpoint
+     *
      * @return direct connection timeout
      */
     public Duration getConnectionTimeout() {
```

Comment:
connection vs socket vs others: Is there guidance or patterns from azure SDK's?

Reply:
So far we have seen connection - as that is the easiest concept for an end user. 
Http client from azure-core -> https://github.com/Azure/azure-sdk-for-java/blob/fc2612029135148ad427083b8a24b672e634a1f6/sdk/core/azure-core-http-jdk-httpclient/src/main/java/com/azure/core/http/jdk/httpclient/JdkAsyncHttpClientBuilder.java#L73
It uses connectionTimeout as well.